### ISSUE TITLE
Refresh time?

### ISSUE THREAD

**Issue by giuliomagnifico** [runezor/PiArtFrame#11]
Hi,
I was just thinking about the refresh time because the script can take a long time to render an image/fractal, and the e-ink display can be damaged by this.

There’s a solution for this issue, or maybe can you add a “clear/refresh” to your code every X seconds (1 min)?

edit: I’m writing this because I damaged my e-ink display…

thanks

**Comment by karlmose** [comment]
Good point. I'm currently writing my first year report, but will aim to make an update within a couple weeks.

**Comment by giuliomagnifico** [comment]
Perfect thanks. Because on the Waveshare website I read:

> When using the e-Paper display, it is recommended that the refresh interval is at least 180s, and refresh at least once every 24 hours. If the e-Paper is not used for a long time, you should use the program to clear the screen before storing it. (Refer to the datasheet for specific storage environment requirements.)

And to rendering a new image a RPi can take 4/6h!

I think it’s quite important, to avoid that your display becomes like mine :) 

![IMG_1892](https://github.com/runezor/PiArtFrame/assets/1975176/c6d47b68-35b1-41c1-be67-074081020616)

**Comment by karlmose** [comment]
Ah that is a sad sight  :( will keep this thread updated
